cmd/agent/subcommands/run/dependent_services_windows.go

```diff
@@ -53,8 +53,9 @@ var subservices = []Servicedef{
 	{
 		name: "sysprobe",
 		configKeys: map[string]config.Config{
-			"network_config.enabled":      config.SystemProbe,
-			"system_probe_config.enabled": config.SystemProbe,
+			"network_config.enabled":                              config.SystemProbe,
+			"system_probe_config.enabled":                         config.SystemProbe,
+			"system_probe_config.windows_crash_detection.enabled": config.SystemProbe,
```

Comment:
I think this is the correct path now, right?
```suggestion
			"windows_crash_detection.enabled": config.SystemProbe,
```

Reply:
yes